ci/acceptance_tests.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
I snuck this in ... just a copy/paste of the original file to get the naming consistent. Will delete original once Jenkins is updated.